src/interactions/scrollInteraction.ts

```diff
@@ -0,0 +1,58 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Interaction {
+
+  export type ScrollCallback = (p: Point, deltaAmount: number) => any;
+
+  export class Scroll extends Interaction.AbstractInteraction {
+
+    /**
+     * The number of pixels occupied in a line.
+     */
+    public static PIXELS_PER_LINE = 120;
```

Comment:
The best I've got is this code from D3
`-d3.event.deltaY * (d3.event.deltaMode ? 120 : 1);`

I'll go look around for a more legitimate source